lte/gateway/c/oai/tasks/nas5g/include/ies/M5GAuthenticationParameterAUTN.h

```diff
@@ -19,6 +19,7 @@ namespace magma5g {
 class AuthenticationParameterAUTNMsg {
  public:
 #define AUTN_MIN_LEN 16
+#define AUTN_MAX_LEN 16

```

Comment:
Unused in this PR, is this being used in a future PR?

Reply:
Its being used in amf